src/nordjylland_news/image_caption_dataset.py

```diff
@@ -97,6 +103,8 @@ def get_image_data(self, article: dict) -> None:
                 if uuid not in self.seen_uuids and caption is not None:
                     data = self._get_image_meta_data(content, article)
                     self.seen_uuids.add(uuid)
+                    if download_images:
+                        self.download_image(data["download_url"], data["file_name"])
```

Comment:
Normalt er det ret god stil at bruge navngivne argumenter, så det er lidt lettere at forstå hvad der sker 🙂 

```suggestion
                        self.download_image(download_url=data["download_url"], file_name=data["file_name"])
```